src/org/traccar/notification/NotificationFormatter.java

```diff
@@ -48,6 +50,9 @@ private static VelocityContext prepareContext(long userId, Event event, Position
             velocityContext.put("geofence", Context.getGeofenceManager().getGeofence(event.getGeofenceId()));
         }
         velocityContext.put("webUrl", Context.getVelocityEngine().getProperty("web.url"));
+        velocityContext.put("dateTool", new DateTool());
+        velocityContext.put("timezone", ReportUtils.getTimezone(userId));
+        velocityContext.put("locale", Locale.getDefault());
```

Comment:
There is no variant of `DateTool.format()` without locale and with timezone, just pass default one.
The side effect is that default locale may be overridden via JVM parameters. And user can format date in desired locale.